### PR TITLE
add a small test for BZ 1009079 and a bit of related refactoring

### PR DESCRIPTION
**Pull request by zkrejcov**

### CODE REVIEW

**Comment by manstis** [comment]
@Rikkola one for you to review please!

**Comment by Rikkola** [comment]
Looks like this change makes the set and get consistent.

+1

**Comment by Rikkola** [comment]
Not to mention we did not test even the set before this.

File: kie-wb-common-widgets/kie-wb-common-ui/src/test/java/org/kie/workbench/common/widgets/client/widget/LiteralTextBoxTest.java [hunk 1]
```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright 2016 JBoss by Red Hat.
```
Comment (yurloc) [inline kie-wb-common-widgets/kie-wb-common-ui/src/test/java/org/kie/workbench/common/widgets/client/widget/LiteralTextBoxTest.java]: @zkrejcov I think you should update your license header template. Not sure how crucial it is but the correct version is `Copyright 2016 Red Hat, Inc. and/or its affiliates.` now.

Comment (zkrejcov) [inline kie-wb-common-widgets/kie-wb-common-ui/src/test/java/org/kie/workbench/common/widgets/client/widget/LiteralTextBoxTest.java]: I wish I knew where it's coming from.. looking. Thanks for the info.

Comment (yurloc) [inline kie-wb-common-widgets/kie-wb-common-ui/src/test/java/org/kie/workbench/common/widgets/client/widget/LiteralTextBoxTest.java]: https://github.com/droolsjbpm/droolsjbpm-build-bootstrap/blob/master/README.md if you haven't found it yet.

**Pull request merged**